### ISSUE TITLE
Not a valid command 'run-main'

### ISSUE THREAD

**Issue by dhakad97** [allenai/pdffigures2#19]
HI,
I am getting error while I run my python script. I am using-
python:3.6
ubuntu:17.10
![a](https://user-images.githubusercontent.com/22168667/35480355-221a96f8-0432-11e8-8dcf-8ef1b31d0e56.png)

**Comment by Sparkier** [comment]
I think changing run-main to runMain already does the trick.

**Comment by JaniceXiong** [comment]
I meet the same error. Did you fix it?
![image](https://user-images.githubusercontent.com/43976565/122406070-26ae8800-cfb3-11eb-97f3-401587bb306e.png)

**Comment by Sparkier** [comment]
Yes, I have this PR: https://github.com/allenai/pdffigures2/pull/33 which is based on this fork: https://github.com/Sparkier/pdffigures2. You should be able to get it running using that.

**Comment by JaniceXiong** [comment]
> Yes, I have this PR: #33 which is based on this fork: https://github.com/Sparkier/pdffigures2. You should be able to get it running using that. It seems like this repo is not maintained anymore, though, so my PR does not get attention.

Thanks a lot! I fixed above problem. But when I change the sbt version to 1.3.0 and use your PR, the new problem occur. Do you have any idea of that? It seems that many dependent package is not found ... @Sparkier 
![image](https://user-images.githubusercontent.com/43976565/122731853-434a0900-d2ae-11eb-9fed-b78556fea7b6.png)

**Comment by dirkraft** [comment]
Hello. A coworker pointed out your activity here. We don't have any bandwidth to offer support for this repo, but I may be able to give you some pointers.

"common-core" refers to a module defined in this repo. https://github.com/allenai/common

Since bintray [shut down](https://jfrog.com/blog/into-the-sunset-bintray-jcenter-gocenter-and-chartcenter/), that might have been where this dependency was previously available, the part about dl.bintray.com/allenai... You might have some luck building/installing the dependency locally of that other repo, but I do not know how far down the rabbit hole goes. There may be other missing transitive dependencies. Cloning that repo and running `sbt +publishLocal` would be the first step. Maybe that will just work for this repo, or maybe you will have to chase down some other dependencies or sbt settings that lets you use locally published dependencies.
